pkg/apis/core/v1alpha1/defaults.go

```diff
@@ -85,6 +85,43 @@ func SetDefaults_VolumeType(obj *VolumeType) {
 	}
 }
 
+// SetDefaults_Seed sets default values for Seed objects.
+func SetDefaults_Seed(obj *Seed) {
+	if obj.Spec.Settings == nil {
+		obj.Spec.Settings = &SeedSettings{}
+	}
+
+	if obj.Spec.Settings.ExcessCapacityReservation == nil {
+		enabled := true
+		for _, taint := range obj.Spec.Taints {
+			if taint.Key == DeprecatedSeedTaintDisableCapacityReservation {
+				enabled = false
+			}
+		}
+		obj.Spec.Settings.ExcessCapacityReservation = &SeedSettingExcessCapacityReservation{Enabled: enabled}
```

Comment:
I'm not really sure what you are suggesting here and why. Do you argue that it is not necessary setting it to `true` because the default (if `.spec.settings==nil` or `.spec.settings.excessCapacityReservation==nil` is anyways `true`)? If yes, then: I still think it's more explicit to set the field as it'll be the new default.

With respect to the argument that it'll help with optimizing the network traffic: Theoretically yes, of course, but I don't think we are at a point yet where we have to worry about these few bytes ;-) If it's really that costly then we can also optimize by syncing resources like the `Seed` only once and not as part of each `Cluster` resource.

--> I would like to stay explicit here if there is no strong reason against it.

Reply:
You got my point.
I am fine with keeping it like this.